src/main/reactjs/src/components/mypage/point/Pointcheck.js

```diff
@@ -30,94 +42,112 @@ const Pointcheck = () => {
 
     const getmember = () => {
         const url = "/member/info?userid=" + storedId;
-        axios.post(url).then((res) => {
-            setmember(res.data);
-        });
-    };
+        axios.post(url)
+            .then(res => {
+                setmember(res.data);
+
+            })
+    }
 
     const fetchPoint = () => {
         setLoading(true);
-        axios
-            .post(`/admin/point?usercode=${usercode}`)
-            .then((res) => {
+        axios.post(`/admin/point?usercode=${usercode}`)
+            .then(res => {
                 setPoint(res.data);
+                setListDisplay(res.data);
             })
-            .catch((error) => {
+            .catch(error => {
                 console.error("포인트 사용내역을 불러오는 중 오류 발생:", error);
             })
             .finally(() => {
                 setLoading(false);
             });
-    };
+    }
 
     const pay5000 = () => {
         setPrice(5000);
-    };
+    }
 
     const pay10000 = () => {
         setPrice(10000);
-    };
+    }
+
 
     const selectpayment = () => {
         ReactSwal.fire({
-            icon: "question",
+            icon: 'question',
             html: `
                     충전할 금액을 선택해 주세요!<br/>
                     <button onclick=${pay5000()}>5000</button>
                     <button onclick=${pay10000()}>10000</button><br/>
                     충전금액 : ${price}
                   `,
             showCancelButton: true,
-            confirmButtonText: "충전",
-            confirmButtonColor: "skyblue",
-            cancelButtonText: "취소",
-        }).then((res) => {
+            confirmButtonText: '충전',
+            confirmButtonColor: 'skyblue',
```

Comment:
```suggestion
            confirmButtonColor: '#5279FD',
```
저희 Swal 확인 버튼은 deepblue 색상으로 통일하게 되어, 다음과 같이 수정해주시기 바라요!